src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -205,13 +205,17 @@ public static IndexReader getReader(String path) throws IOException {
 
   public static Map<String, Long> getTermCounts(IndexReader reader, String termStr) throws IOException, ParseException {
     EnglishAnalyzer ea = new EnglishAnalyzer(CharArraySet.EMPTY_SET);
```

Comment:
hey, while you're at it, can you refactor to our new way of creating analyzer instances?